shared/models.go

```diff
@@ -99,14 +99,16 @@ type RulesMatched struct {
 }
 
 func (r *RulePolicy) validateType() bool {
-	permitedTypes := []string{"json", "header", "latency"}
+	permitedTypes := []string{"json", "header", "slo"}
 	_, found := Find(permitedTypes, r.Type)
 	if !found {
-		fmt.Printf("\nRule with name %s will be ignored. Err: only json, header and latency types are supported on rule definition.\n", r.Name)
+		log.Printf("Error: %s. ", r.Name)
+		log.Fatalf("Only json, header and slo types are supported on rule definition.\n")
 	}
-	if strings.ToLower(r.Type) == "latency" {
-		if r.Latency == 0 {
-			fmt.Printf("\nRule with name %s will be ignored. Err: when type=latency, the field Latency should be specified and have a value >= 1\n\n", r.Name)
+	if strings.ToLower(r.Type) == "slo" {
+		if r.ResponseTime == 0 {
+			log.Printf("Error: %s. ", r.Name)
+			log.Fatalf("When type=slo, the field response-time should be specified and have a value >= 1\n\n")
```

Comment:
I guess this is not something ready for merge (Or I don't understand why do we have here log.Fatal...)

Reply:
Now i'm printing that the rule will be ignored